BaseTools/Source/C/BrotliCompress/GNUmakefile

```diff
@@ -10,7 +10,11 @@ APPNAME = BrotliCompress
 
 OBJECTS = \
   BrotliCompress.o \
+  brotli/c/common/constants.o \
+  brotli/c/common/context.o \
   brotli/c/common/dictionary.o \
+  brotli/c/common/platform.o \
+  brotli/c/common/shared_dictionary.o \

```

Comment:
I manually added the new object files here. Is there a script that should be used to update these automatically or was this the right way to do it?